external-crates/move/crates/move-compiler/src/hlir/ast.rs

```diff
@@ -245,6 +247,14 @@ pub enum LValue_ {
 }
 pub type LValue = Spanned<LValue_>;
 
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub enum AssignCase {
```

Comment:
I would have preferred having two different `Assign` commands, but I appreciate how much more work it would be to swap out everywhere.

Reply:
I can make two different assign commands if you would prefer. But this seemed preferable not just because of the difficulty to refactor, but also since 90% of the time, you don't care about the case.

Again, happy to swap it out if you think it will be better long term